nixos/modules/services/audio/spotifyd.nix

```diff
@@ -23,6 +33,13 @@ in
   };
 
   config = mkIf cfg.enable {
+    assertions = [
+      {
+        assertion = cfg.config == "" || cfg.settings == {};
+        message = "Using the stringly typed .config attribute is discouraged. Use the TOML typed .settings attribute instead.";
+      }
+    ];
+
```

Comment:
Should the error message not be something like "at most one of the .config attribute and the .settings attribute may be set"?